types/meteor/mongo.d.ts

```diff
@@ -83,7 +83,7 @@ declare module 'meteor/mongo' {
             $query: Query<T>;
             $comment?: string | undefined;
             $explain?: any;
-            $hint?: any;
+            $hint?: Hint;
```

Comment:
It's typed as `Hint` here but in `types/mongo.d.ts` it's typed as `unknown` - is it intended?